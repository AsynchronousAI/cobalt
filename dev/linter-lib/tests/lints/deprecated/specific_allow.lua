deprecated_allowed()
deprecated_blocked()

more.deprecated_allowed()
wow.extra.deprecated_allowed()

deprecated_allowed.more()
