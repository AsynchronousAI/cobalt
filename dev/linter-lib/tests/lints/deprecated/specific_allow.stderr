error[deprecated]: standard library function `deprecated_blocked` is deprecated
  ┌─ specific_allow.lua:2:1
  │
2 │ deprecated_blocked()
  │ ^^^^^^^^^^^^^^^^^^^^
  │
  = this is deprecated

